test_api/04_test_updating_resources.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from test_api.constant import TestData
+
+
+@pytest.mark.parametrize('todos_id', [TestData.TODOS_VALUE, (TestData.TODOS_VALUE - 1), (TestData.TODOS_VALUE - 50)])
+def test_update_resources_put_positive(create_session, workplace, todos_id):
+    response = create_session.put(url=f'{workplace}/{todos_id - 1}', json=TestData.DICT_UNIT_1)
```

Comment:
I guess it's better to use **1** and **TestData.TODOS_VALUE** todos ids here:

```
@pytest.mark.parametrize('todos_id', [1, TestData.TODOS_VALUE])
def test_update_resources_put_positive(create_session, workplace, todos_id):
    response = create_session.put(url=f'{workplace}/{todos_id}', json=TestData.DICT_UNIT_1)
```

Reply:
added